projectgenerator/libili2db/ili2dbconfig.py

```diff
@@ -115,14 +125,18 @@ def __init__(self):
         self.ilimodels = ''
         self.tomlfile = ''
 
-    @property
-    def uri(self):
+    def uri(self, su = False):
```

Comment:
API docs for `su` are missing.
Another approach would be a property `super_user_uri`.

Reply:
Just for information or should a second property be preferred?

I decided for method because of avoiding to have double written code here. But I don't really mind.

Reply:
Another property would be nice to have cleaner code. To avoid duplicate code, this method can be called internally.

Reply:
You mean two properties calling the method? 
Okay, I can do that...

Reply:
Yes, and the method prefixed with a `_` to mark it as private and a short API doc on top to explain the parameter.